src/main/kotlin/org/opensearch/commons/alerting/model/DocLevelQuery.kt

```diff
@@ -160,15 +161,20 @@ data class DocLevelQuery(
         }
 
         // TODO: add test for this
```

Comment:
Do you want to address this as part of this PR?

Reply:
This PR modifies the existing test in `DocLevelMonitorInputTests`, should we add more testing for this? If not I will remove this comment.
https://github.com/opensearch-project/common-utils/blob/150413f3e0b2c97f39a95ca997eb9fc433df4ab3/src/test/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInputTests.kt#L45

Reply:
If there is no test for the part of the code that the TODO was added for, then let's add one?

Reply:
These two unit tests should be sufficient to test the two validation functions. I think the TODO may have been left in after the tests were created. These two tests create a doc level monitor with an invalid doc level query so that validation for a doc level query name and doc level query tag exceptions are caught.

1. https://github.com/opensearch-project/common-utils/blob/aa2b6c8fbfa1e6f3ffe967d8503b162677f54ace/src/test/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInputTests.kt#L45
2. https://github.com/opensearch-project/common-utils/blob/aa2b6c8fbfa1e6f3ffe967d8503b162677f54ace/src/test/kotlin/org/opensearch/commons/alerting/model/DocLevelMonitorInputTests.kt#L65

Will take out the TODO comment